src/SocialMediaDashboard.Common/DTO/UserDTO.cs

```diff
@@ -0,0 +1,47 @@
+﻿using SocialMediaDashboard.Common.Interfaces;
+using System.Text.Json.Serialization;
+
+namespace SocialMediaDashboard.Common.DTO
+{
+    /// <summary>
+    /// UserDTO model.
+    /// </summary>
+    public class UserDTO : IHasDbIdentity
+    {
+        /// <summary>
+        /// Identifier.
+        /// </summary>
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Email.
+        /// </summary>
+        public string Email { get; set; }
+
+        /// <summary>
+        /// Password.
+        /// </summary>
+        [JsonIgnore]
+        public string Password { get; set; }
```

Comment:
Better save password hash in db instead of password

Reply:
Fix it.